SqlQueryBuilder/Select/SelectQueryBuilder.cs

```diff
@@ -18,6 +18,7 @@ public class SelectQueryBuilder : IQueryBuilderSelectFrom, IQueryBuilderJoinOrSe
         private List<string> JoinClauses = new List<string>();
         private List<string> OrderByClauses = new List<string>();
         private List<string> GroupByClauses = new List<string>();
+        private List<int> SkipTakeClauses = new List<int>();
```

Comment:
Could use a better type to store the skip and take values, maybe a ValueTuple ? ( int, int )
```suggestion
        private List<int> SkipTakeClauses = new List<int>();
        // becomes
        private (int skip, int take) SkipTake =  (0, 0);
```

Reply:
`IQueryBuilderOrderBy Skip(int skip);
 IQueryBuilderOrderBy Take(int take);`
 I seperated into `Skip` and `Take` two different methods.

Reply:
Perfect!